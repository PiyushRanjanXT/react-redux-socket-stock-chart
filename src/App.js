import * as React from "react";
import { Provider } from "react-redux";
import { BrowserRouter as Router, Route, Redirect } from "react-router-dom";
import "bootstrap/dist/css/bootstrap.min.css";

import Menu from "./components/menu/";
import store from "./store";
import Dashboard from "./modules/dashboard";
import LiveChart from "./modules/live-chart";

const App = () => (
  <Provider store={store}>
    <Router>
      <Menu></Menu>
      <div className="container">
        <Route exact path="/">
          <Redirect to="/dashboard" />
        </Route>
        <Route path="/dashboard" component={Dashboard} />
        <Route path="/chart" component={LiveChart} />
      </div>
    </Router>
  </Provider>
);

export default App;
