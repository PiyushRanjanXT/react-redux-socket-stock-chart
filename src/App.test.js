import React from "react";
import { shallow } from "enzyme";
import App from "./App";

describe("App", () => {
  it("Should contain container class", () => {
    const component = shallow(<App />);

    expect(component.find(".container")).toHaveLength(1);
  });

  it("Screenshot should be matched", () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
