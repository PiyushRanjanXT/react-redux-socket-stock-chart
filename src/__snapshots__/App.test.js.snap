// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Screenshot should be matched 1`] = `ShallowWrapper {}`;

exports[`MyComponent should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
